data/tlg4034/tlg006/tlg4034.tlg006.opp-grc1.xml

```diff
@@ -95,15 +95,17 @@
   <profileDesc>
    <langUsage>
     <language ident="grc">Greek</language>
-    <language ident="lat">Latin</language>
```

Comment:
The "Latin" `<language>` should not be deleted since the `xml:lang="lat"` attribute is used in the teiHeader.
Also, `<language ident="deu">German</language>` element should be added.